ivy_tests/test_ivy/test_functional/test_core/test_statistical.py

```diff
@@ -45,7 +45,7 @@ def _statistical_dtype_values(draw, *, function, min_value=None, max_value=None)
         small_abs_safety_factor = 24
     dtype, values, axis = draw(
         helpers.dtype_values_axis(
-            available_dtypes=helpers.get_dtypes("valid"),
+            available_dtypes=helpers.get_dtypes("float"),
```

Comment:
recently we've moved towards generating `"valid"` dtypes instead of `"float"` for every test. Ideally the unsupported dtype decorator should prevent all unintended dtypes from being generated (like `bool` or `int`)